src/components/MetricsPane/charts/AggregateFishBiomas.jsx

```diff
@@ -13,7 +13,7 @@ export const AggregateFishBiomass = () => {
 
   const surveyFishbeltBiomassValues = filteredSurveys
     .map((record) => record.protocols?.beltfish?.biomass_kgha_avg)
-    .filter(Boolean)
+    .filter((record) => record !== undefined && record !== null)
```

Comment:
why did you change this?

Reply:
`0` should be counted

Reply:
Lol, the most obvious thing I didnt think of!!! <3